### ISSUE TITLE
getting sidetip plugin not defined error 

### ISSUE THREAD

**Issue by flyingboy007** [Amunds/tooltipster-rails#9]
Trying to integrate this gem but getting below error.  Added all these 
```
//= require tooltipster.bundle
//= require tooltipster.bundle.min
//= require tooltipster.main
//= require tooltipster.main.min
```

> tooltipster.main.min.self-697531f….js?body=1:2 Uncaught Error: The "sideTip" plugin is not defined
>     at a.Tooltipster._plug (tooltipster.main.min.self-697531f….js?body=1:2)
>     at String.<anonymous> (tooltipster.main.min.self-697531f….js?body=1:2)
>     at Function.each (jquery.self-bd7ddd3….js?body=1:371)
>     at a.Tooltipster.__init (tooltipster.main.min.self-697531f….js?body=1:2)
>     at a.Tooltipster (tooltipster.main.min.self-697531f….js?body=1:2)
>     at HTMLElement.<anonymous> (tooltipster.main.min.self-697531f….js?body=1:2)
>     at Function.each (jquery.self-bd7ddd3….js?body=1:371)
>     at jQuery.fn.init.each (jquery.self-bd7ddd3….js?body=1:138)
>     at jQuery.fn.init.a.fn.tooltipster (tooltipster.main.min.self-697531f….js?body=1:2)
>     at general:267
> ﻿
> 
> ​

What am doing wrong here?

**Comment by flyingboy007** [comment]
I should have read the docs. Sorry just noticed that only one is needed. Thank you

**Comment by meowsus** [comment]
Hey @flyingboy007,

I'm sorry you're having trouble. When I saw your message I thought for sure that this was due to the latest release I pushed for this gem (v4.2.3) so I spun up a test app to try and reproduce.

I was able to confirm that the latest release seems to be working fine with:

```js
//= require tooltipster.bundle

$(document).ready(function() {
    $('.tooltip').tooltipster();
});
```

appended to my `application.js` manifest and:

```scss
 *= require tooltipster.bundle
 *= require plugins/tooltipster/sideTip/themes/tooltipster-sideTip-light.min
```

appended to my `application.css` manifest.
